libvast/test/pipeline.cpp

```diff
@@ -163,6 +162,68 @@ concatenate(caf::expected<std::vector<vast::table_slice>> slices) {
 
 FIXTURE_SCOPE(pipeline_tests, pipelines_fixture)
 
+TEST(head 1) {
```

Comment:
There should be unit tests that cover using `head` and `taste` with numbers <= 0. If the pipeline parsing should fail at that point (or in the case of 0, show nothing) , it should be explicitly proven in these unit tests.

Reply:
The parser will fail with negative numbers, I can add a test for this. Is `head 0` or `taste 0` really an error, though? The current behavior is that it just always yields nothing.

Reply:
`head 0 ` is not an error to me, it just shows nothing (which is logically correct behavior), ie. `results.empty()`.
It just helps to have such cases specified in unit tests.

Reply:
Okay, will add them.

Reply:
I believe I covered everything in https://github.com/tenzir/vast/pull/2891/commits/916060881347d0e5a0978c185e4c3f068312bc85 – mind taking another look?

Reply:
Yes, it's all good now. Thank you.